packages/react-components/react-carousel-preview/stories/src/Carousel/CarouselBestPractices.md

```diff
@@ -2,4 +2,15 @@
 
 ### Do
 
+- Limit cards to a max width of 100% of screen size.
+- Use 'gap' css on CarouselSlider to provide easy spacing between cards, use margin if circular is enabled.
+- Recommended to keep cards a uniform size where possible.
```

Comment:
Consider rephrasing for active tone:

"Keep cards a uniform size where possible"